### DISCUSSION TITLE
Add more Track Designs

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16917] [Feature requests]
Would the developers consider adding more prebuilt rides to the game?

pros : 
small filesize, help new players, showcase new ride types / elements, showcase community members, teach good layouts, emphasize blueprint format when ready

 cons : may not feel authentic, potential for bloat, maybe a submission backlog or argument, may add to compile time, may cause compatibility issues in the future

**Comment by IntelOrca** [comment]
Could do, but we would probably wait for the new track design format first, so that new features could be leveraged in the offical OpenRCT2 track / scenery designs.
